plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/inner-blocks/review-form/style.scss

```diff
@@ -0,0 +1,83 @@
+// Allow these default styles to be overridden by global styles.
+:where(.wp-block-woocommerce-product-review-form) {
```

Comment:
A lot of these styles are mimicked from the comments form block, with the class names updated where necessary.